### PR TITLE
Mergedinitstyleguide

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: clientkit2/default-stylesheets.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+stylesheets:
```
Comment (dawnerd) [inline clientkit2/default-stylesheets.yaml]: What's `clientkit2`?

Comment (orthagonal) [inline clientkit2/default-stylesheets.yaml]: Let me pull that out, it shouldn't have gotten into the PR.

File: app.js [hunk 2]
```diff
@@ -45,133 +52,45 @@ const argv = yargs
...
-    conf.mode = 'prod';
-  }
-  return conf;
-};
 
 if (argv.styleguide) {
-  // todo: this should ultimately go in commands/ instead of tasks/ on the init branch:
-  const config = loadConfig();
-  const styleguide = require('./tasks/styleguide.js');
+  const conf = configHandler.loadConfig(defaultConf, argv, log);
```
Comment (jgallen23) [inline app.js]: we can kill this command since we are auto running on yaml change

File: commands/init.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+'use strict';
+const mkdirp = require('mkdirp');
+const path = require('path');
+const Logr = require('logr');
+const fs = require('fs');
+const log = new Logr({
+  type: 'cli',
+  renderOptions: {
+    cli: {
+      lineColor: 'cyan'
```
Comment (jgallen23) [inline commands/init.js]: lets change this color so it's different than the others

File: lib/watcher.js [hunk 4]
```diff
@@ -11,26 +11,35 @@ const log = new Logr({
...
+    log(['error'], error);
+  });
+  if (Object.keys(taskList).length > 0) {
+    watcher.on('all', debounce(() => {
+      for (const file of files) {
+        processor(file, taskList[file]);
+      }
+    }), delay);
+  } else {
+    watcher.on('all', debounce(() => {
```
Comment (jgallen23) [inline lib/watcher.js]: whats this use case?

**Comment by jgallen23** [comment]
need to merge in from master

**Pull request merged**